.gitignore

```diff
@@ -47,6 +47,4 @@ buck-out/
 .npmrc
 
 # ignore compiled
-**/*.js
-**/*.d.ts
```

Comment:
what is the reason of removing these?

Reply:
@mieszko4 I am sorry you can ignore my changes in here. It's suddenly change I don't know. my changes is only in `RNZoomUsModule.java`

Reply:
I cannot just ignore it when merging pr.